src/controller/data_model/controller-clusters.zap

```diff
@@ -735,6 +735,103 @@
             }
           ]
         },
+        {
+          "name": "Binary Input (Basic)",
```

Comment:
Can you remove the `Silabs` definitions from `src/app/zap-templates/zlc/data_model/Silabs/general.xml` and `src/app/zcl/zap-templates/data_model/Silabs/general-thread.xml` and add your own into `src/app/zap-templates/zcl/input-output-value.xml` ? It will be easier to compare the definition to the spec.

Reply:
Sure I can do this. The only think is that I don't know which side effects this introduces. For example `src/app/zap-templates/zcl/data-model/silabs/ha-devices.xml` does reference the Binary Input Basic cluster.